security/rbac/docs/index.rst

```diff
@@ -0,0 +1,461 @@
+.. _rbac_demo:
+
+RBAC Demo
+=========
+
+This demo showcases the |rbac-long| functionality in |cp|.
+It is mostly for reference to see a workflow using the new |rbac| feature across the services in |cp|.
+
+There are two ways to run the demo.
+
+-  :ref:`rbac_demo_local`
+-  :ref:`rbac_demo_docker`
+
+
+.. _rbac_demo_local:
+
+===============================================
+Run demo on local install of Confluent Platform
+===============================================
+
+This method of running the demo is for users who have `downloaded <https://www.confluent.io/download/>`__ |cp| to their local hosts.
+
+Caveats
+-------
+
+-  For simplicity, this demo does not use LDAP, instead it uses the Hash
+   Login service with statically defined users/passwords. Additional
+   configurations would be required if you wanted to augment the demo to
+   connect to your LDAP server.
+-  The |rbac| configurations and role bindings in this demo are not
+   comprehensive, they are only for development to get minimum |rbac|
+   functionality set up across all the services in |cp|.
+   Please refer to the :ref:`RBAC documentation <rbac-overview>`
+   for comprehensive configuration and production guidance.
+
+Prerequisites
+-------------
+
+* :ref:`Confluent CLI <cli-install>`: |confluent-cli| must be installed on your machine, version ``v0.127.0`` or higher (note: as of |cp| 5.3, the |confluent-cli| is a separate :ref:`download <cli-install>`).
+
+* `Confluent Platform 5.3 or higher <https://www.confluent.io/download/>`__: use a clean install of |cp| without modified properties files, or else the demo is not guaranteed to work.
+
+* jq tool must be installed on your machine.
+
+Run the demo
+------------
+
+#. Clone the `confluentinc/examples <https://github.com/confluentinc/examples>`__ repository from GitHub and check out the :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+       git clone git@github.com:confluentinc/examples.git
+       cd examples
+       git checkout |release_post_branch|
+
+#. Navigate to ``security/rbac/scripts`` directory.
+
+   .. codewithvars:: bash
+
+       cd security/rbac/scripts
+
+#. You have two options to run the demo.
+
+-  Option 1: run the demo end-to-end for all services
+
+   .. code:: bash
+
+      ./run.sh
+
+-  Option 2: step through it one service at a time
```

Comment:
the indentation is messed up here, resetting the numbering.

![image](https://user-images.githubusercontent.com/11722533/68782969-13755280-05ef-11ea-9d26-94feae370d89.png)

Reply:
@joel-hamill fixed in https://github.com/confluentinc/examples/pull/329/commits/16922e70ed50351e7e9de662b80eb52e18f4d414